omero/sysadmins/troubleshooting.txt

```diff
@@ -342,6 +344,29 @@ OMERO.web did not start
 
      :ome-users:`upgrade 5.0.5 to 5.1.1 omero.web forbidden <2015-April/005316.html>`
 
+OMERO.web running but status says not started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you upgraded OMERO but forgot to stop OMERO.web, processes will still be
+running. In order to kill stale processes by hand, run:: $ ps aux | grep /home/omero/OMERO.server/var/django.pid
```

Comment:
This command isn't marked-up on staging because you need to start a new line and indent for the `::` to work